packages/pytest-simcore/src/pytest_simcore/helpers/s3.py

```diff
@@ -29,7 +30,8 @@ async def _file_sender(
             num_read_bytes += len(chunk)
             yield chunk
             if raise_while_uploading:
-                raise RuntimeError("we were asked to raise here!")
+                msg = "we were asked to raise here!"
```

Comment:
Not sure I like the `msg` too much, but c’est la vie I guess... 🤷

Reply:
:rofl: it is a test, it's ok ;)